tests/test_traits.py

```diff
@@ -343,3 +348,70 @@ def test_filter_value_validation_filter_size_3():
             np.array([1, 2, 3, 4, 5, 6, 7, 8, 9], dtype=np.float64), 3
         ),
     )
+
+
+class NormalAccessorWidget(BaseLayer):
+    _rows_per_chunk = 2
+
+    table = pa.table({"data": [1, 2, 3]})
+
+    value = NormalAccessor()
+
+
+def test_normal_accessor_validation_list_length():
+    # tuple or list must have 3 elements
+    with pytest.raises(TraitError):
+        NormalAccessorWidget()
+
+    with pytest.raises(TraitError, match="expected 3 values if passed a tuple or list"):
```

Comment:
The `match` here is tested against the error string, so if you're copying tests from above you might need to adjust the expected error string, or use the same error message in your code.

Reply:
Yes I will update these to match